### DISCUSSION TITLE
Heightmap resolution based on geometry and zoom lvl.

### DISCUSSION THREAD

**Discussion by Yberon** [domlysz/BlenderGIS#705] [General]
Hello, 
Why heightmap resolution is based on zoom level? 
i.e. When i generate zoomed **(with L lock)** terrain it's more detailed than bigger one. 
Shouldn't heightmap resolution be locked on it's  limit (90m) all the time?: **[larger area=bigger heightmap resolution]**? 
Larger areas are almoust no detiled because of low geometry of the plane -further subdividing does not help at all.
How to create bigger area in that case? Generate small tiles and stitch them?

**Comment by AnotherOneOleg** [comment]
I also do not understand how to make the mesh density higher. The mesh density is always the same even when I take a huge area that includes the city and the surrounding mountains.
